lib/ibandit/local_details_cleaner.rb

```diff
@@ -330,9 +330,7 @@ def self.clean_hr_details(local_details)
     def self.clean_hu_details(local_details)
       # This method supports being passed the component IBAN parts, as defined
       # by SWIFT, or a single 16 or 24 digit string.
-      if local_details[:bank_code] || local_details[:branch_code]
-        return local_details
-      end
+      return local_details if local_details[:bank_code] || local_details[:branch_code]
```

Comment:
Was this change required by rubocop? I think it's much less clear than the original, I would need to remember the precedence of `if` vs `||` to understand what is returned

Reply:
I agree, tried to put brackets around the `||` but Rubocop wasn't having it